install.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+for main in index.{"/usr/bin/git","/usr/bin/unzip","/usr/bin/wget"}
+do
+    command -v "$main" || deps=false
+done
+
+function install-theme()
+{
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Get the latest release of SFP and unzip it
```

Comment:
Entirely not your fault, but it turns out SFP isn't actually required-- you can safely remove all SFP-related code!